.github/ISSUE_TEMPLATE/V&V_request.md

```diff
@@ -62,25 +66,29 @@ Note: not applicable items will be struck through
 
 *Ensure all above items have been completed before continuing*
 
-@biosensics/test-and-manufacturing :
+**@biosensics/test-and-manufacturing :**
 - [ ] V&V executed
 - [ ] Completed V&V submitted to [Quality](mailto:quality@biosensics.com)
 
-@biosensics/engineering :
-- [ ] App ECO Section II submitted to [Quality](mailto:quality@biosensics.com)
-- [ ] App ECO Section II approved
+**@biosensics/engineering :**
+
+Server:
 - [ ] Server ECO Section II submitted to [Quality](mailto:quality@biosensics.com)
 - [ ] Server ECO Section II approved
 - [ ] Ensure correct Zoom meeting room is set
 - [ ] Create server release
 - [ ] Deploy server release to Aptible production server
 - [ ] Confirm any updated Aptible production server configuration values
 - [ ] Merge `release` branch into `master` on server repository
```

Comment:
```suggestion
- [ ] Merge `release` branch into `develop` on server repository
```
Should be develop branch, since release would've already been merged into master when the server release is created